charts/nr-ebpf-agent/values.yaml

```diff
@@ -23,6 +23,11 @@ protocols:
   cass: true
   dns: true
   pgsql: true
+# -- The periodicity in seconds at which the eBPF client pushes data to the OTel collector for export to NR.
```

Comment:
minor but should it be this?
```suggestion
# -- The periodicity in seconds at which the eBPF agent pushes data to the OTel collector for export to NR.
```

Reply:
Applied. A little ambiguous from a customer perspective (since it is marketed as a single ebpf "agent") but technically correct.